scripts/controlnet.py

```diff
@@ -189,56 +189,56 @@ def refresh_all_models(*inputs):
                             gr.update(label="Threshold B", value=64, minimum=64, maximum=1024, interactive=False),
                         ]
                     elif module in ["openpose", "openpose_hand", "segmentation"]:
                         return [
-                            gr.update(label="Annotator Resolution", minimum=128, maximum=1024, value=512, step=1, interactive=True),
+                            gr.update(label="Annotator Resolution", minimum=64, maximum=2048, value=512, step=1, interactive=True),
                             gr.update(label="Threshold A", value=64, minimum=64, maximum=1024, interactive=False),
                             gr.update(label="Threshold B", value=64, minimum=64, maximum=1024, interactive=False),
                         ]
                     elif module == "depth":
                         return [
-                            gr.update(label="Midas Resolution", minimum=128, maximum=1024, value=384, step=1, interactive=True),
+                            gr.update(label="Midas Resolution", minimum=64, maximum=2048, value=384, step=1, interactive=True),
                             gr.update(label="Threshold A", value=64, minimum=64, maximum=1024, interactive=False),
                             gr.update(label="Threshold B", value=64, minimum=64, maximum=1024, interactive=False),
                         ]
                     elif module == "normal_map":
                         return [
-                            gr.update(label="Normal Resolution", minimum=128, maximum=1024, value=512, step=1, interactive=True),
+                            gr.update(label="Normal Resolution", minimum=64, maximum=2048, value=512, step=1, interactive=True),
                             gr.update(label="Normal background threshold", minimum=0.0, maximum=1.0, value=0.4, step=0.01, interactive=True),
                             gr.update(label="Threshold B", value=64, minimum=64, maximum=1024, interactive=False),
                         ]
                     elif module == "none":
                         return [
-                            gr.update(label="Normal Resolution", value=64, minimum=64, maximum=1024, interactive=False),
+                            gr.update(label="Normal Resolution", value=64, minimum=64, maximum=2048, interactive=False),
                             gr.update(label="Threshold A", value=64, minimum=64, maximum=1024, interactive=False),
                             gr.update(label="Threshold B", value=64, minimum=64, maximum=1024, interactive=False),
                         ]
                     else:
                         return [
-                            gr.update(label="Annotator resolution", value=512, minimum=64, maximum=1024, step=1, interactive=True),
+                            gr.update(label="Annotator resolution", value=64, minimum=64, maximum=2048, interactive=False),
```

Comment:
Use `value=512` and `interactive=True` for any other preprocessors might be better